test/helpers/utils.js

```diff
@@ -99,7 +100,32 @@ const giveEth = (toAccount, percentage, fromAccounts) => {
   });
 };
 
+const onlyWhitelisted = async (config, ifWhiteListed) => {
```

Comment:
this doesn't need to be async. And it doesn't need to take a callback function.

Reply:
? it's taking a function of which to execute, as long as the account is able to do it. Can you propose a more clear solution?